sdk/formrecognizer/azure-ai-formrecognizer/tests/test_compose_model.py

```diff
@@ -37,6 +37,10 @@ def test_compose_model_with_model_name(self, client, formrecognizer_storage_cont
         self.assertEqual(composed_model_dict.get("model_name"), "my composed model")
         self.assertIsNotNone(composed_model_dict.get("model_id"))
 
+        composed_model = _models.CustomFormModel.from_dict(composed_model_dict)
+        self.assertEqual(composed_model.model_name, "my composed model")
+        self.assertComposedModelHasValues(composed_model, model_1, model_2)
```

Comment:
Recommend switching the from_dict test for CustomFormModel to one that uses the assertModelTransformCorrect helper function so we can compare output against the generated model. Not sure there is a transform test for composed models, but we should have one in the test_training.py